### DISCUSSION TITLE
Details about the initial submission

### DISCUSSION THREAD

**Discussion by shangyihao** [snap-stanford/ogb#161] [MAG240M-LSC]
Hi, i wanna check whether the only thing we can submit that are relevant to the program is the saved model’s prediction with your package, and the only feedback result is the performance i.e. the acc in dataset mag240m ,right?

Thanks!

**Comment by weihua916** [comment]
Yes, that's correct. We will make the submission page public right after the team registration deadline, so stay tuned!

**Reply by shangyihao** [reply]
Therefore, the only criterion for the final selection is performance ?There are no hardware resources or time constraints in dataset mag240m-lsc?

**Reply by weihua916** [reply]
Correct. There are no hardware constraints for mag240m.

**Reply by shangyihao** [reply]
Thanks a lot!

**Reply by zbn123** [reply]
> Correct. There are no hardware constraints for mag240m.

![image](https://user-images.githubusercontent.com/22554360/116644231-6985b380-a9a5-11eb-80f2-cb55c1255f90.png) 
Is there a time limit and hardware limit for mag240m-lsc

**Reply by weihua916** [reply]
No time limit and no hardware limit for mag240m. We just wanted to collect the information, but there is no constraint.

**Reply by zbn123** [reply]
Thanks
